tests/online_migration/sle12_online_migration/snapper_rollback.pm

```diff
@@ -26,10 +26,22 @@ sub check_rollback_system() {
 sub run() {
     my $self = shift;
 
-    # login to snapshot and perform snapper rollback
-    assert_screen 'linux-login', 200;
+    # login to before online migration snapshot
+    # tty would not appear quite often after booting snapshot
```

Comment:
why shouldn't it appear? Is this a potential product issue? IMHO the tests should not blindly hit keys up to six times if nothing happens. At least call `record_soft_failure` with a reference to a bug.